cds_ils/literature/serializers/json.py

```diff
@@ -24,6 +26,7 @@ def transform_record(self, pid, record, links_factory=None, **kwargs):
         eitems = literature["metadata"].get("eitems", {}).get("hits", [])
         for eitem in eitems:
             format_login_required_urls(eitem)
+        field_cover_metadata(literature["metadata"])
```

Comment:
oups it is me this one, sorry I didn't see it in `invenio-app-ils`, and I relied only on tests :(
Thanks for fixing it...